modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+You can customize and configure an existing project access roles name by editing the YAML code, and add an `console.openshift.io/display-name: Display name` annotation to the cluster.
+
+.Procedure
+To customize the cluster role options of all projects that are shown in the *Project Access* tab:
+
+. Navigate to the *Search* view.
+. In the *Search* view, select the *Resources* drop down and search *Console*.
+. Select the *Console `operator.openshift.io/v1`* to add as a new row in resource list.
++
+.Searching Console resource
+image::odc_cluster_console.png[]
+. Select *cluster* under *Name* list.
+. Navigate to *YAML* tab to view and edit the YAML code.
+. In the YAML code under `spec` add or edit the list of `availableClusterRoles` to configure the project roles.
```

Comment:
```suggestion
. In the YAML code under `spec` add or edit the list of `availableClusterRoles`.
```
If I understand it correctly, this whole procedure is about configuring/customizing project roles, so repeating it inside the procedure isn't necessary maybe?

Reply:
That's a good point too. I'll try to rephrase.